src/components/RequestsFilters/RequestsFilters.test.js

```diff
@@ -317,4 +322,60 @@ describe('RequestsFilters', () => {
       expect(RequestLevelFilter).toHaveBeenCalledWith(expect.objectContaining(expectedProps), {});
     });
   });
+
+  describe('Print status accordion', () => {
```

Comment:
Could you please add tests for the behavior 'when isViewPrintDetailsEnabled is not enabled'

Reply:
test case for the behavior 'when isViewPrintDetailsEnabled is not enabled' is covered already. I will move it to separate describe for better readability.
![image](https://github.com/user-attachments/assets/e6537fda-3760-4dd2-b044-9f20c1a2d596)